README.md

```diff
@@ -1,177 +1,164 @@
 
-[Check this out to know more about Red Hat Marketplace](https://marketplace.redhat.com/en-us/about).
-
-### How to install, setup & get started using the Findability Platform Predict Plus operator on Red Hat OpenShift cluster
-
-In this tutorial, we demonstrate how to install, setup and get started using the Findability Platform Predict Plus (FPPredict Plus) operator from Red Hat Marketplace (RHM). Every operator on Red Hat Marketplace has different setup for installation and configuration and the focus of this tutorial is Findability Platform Predict Plus operator.
-
-The advantages of using RHM operators are per below.  
-
-* `Software for any cloud` :- Enterprise software for container-based environments in public clouds and on-prem
-
-* `Automated deployment` :- Fast, integrated experience with instant availability on Red Hat® OpenShift® clusters
-
-* `Fully supported` :- Free, continuous support for products purchased through Red Hat Marketplace
-
-### A brief about Findability Platform Predict Plus operator (Version - 0.0.4)
-
-FP-Predict +™, an Automated, self learning, and Multi Modeling Artificial Intelligence (AI) that handles Discrete Target variable, Continuous Target variable and Time series data with no need for coding. This operator will be helpful to solve some of the usecases under AI using different methods like Regression, Classification, Timeseries etc. This operator will be applicable to developers, data scientists, architects who wants to solve different usecases under machine learning and AI. 
+### Quick intro to Findability Platform Predict Plus operator (Version - 0.0.4)
 
+FP-Predict +™ is an automated, self learning, multi-modeling artificial intelligence (AI) tool that handles discrete target variables, continuous target variables, and time series data with no need for coding. <--EM: WHat is "no need for coding" referncing? Is it just a GUI that you can use?-->
 
 # Prerequisites
 
-* For all operators being installed from RHM, OpenShift cluster version 4.3 or higher is mandatory. Please set up Classic cluster using the instructions from below URL.
+* Red Hat OpenShift version 4.3 is requried to use with the software in Red Hat Markeplace. You can set up a class cluster using these instructions: 
+[Set up OpenShift Cluster](https://cloud.ibm.com/docs/openshift?topic=openshift-getting-started) <!--EM WHy are we sending them to our docs? This is just about how to use Red Hat OpenShift on IBM Cloud-->
 
-[Setting up OpenShift Cluster](https://cloud.ibm.com/docs/openshift?topic=openshift-getting-started)
```

Comment:
It is about setting up the OS cluster on IBM cloud and use it with one of the operators on RHM. It should remain as is.

Reply:
I only moved it, not removed it.